<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QPushButton[plan='true'] {
	background: rgba(255, 255, 255, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="get_combobox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>322</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QComboBox" name="post_combobox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>322</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_post">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отдаете</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accept_top_courses">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Подобрать лучшие обменники</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_get">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Получаете</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_valute">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>291</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество отдаваемой валюты</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="count_valute">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>292</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="maximum">
     <double>100000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="status_answer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Статус возможности обмена &lt;/p&gt;&lt;p&gt;в обменниках&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="answer_marks">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Указывать отношение положительных&lt;/p&gt;&lt;p&gt; к отрицательным отзывам?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_marks">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>295</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="main_icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>52</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Подбор обменников</string>
    </property>
   </widget>
   <widget class="QFrame" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>741</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="source_code">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>59</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Github</string>
    </property>
   </widget>
   <widget class="QLabel" name="source_site_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Исходный сайт</string>
    </property>
   </widget>
   <widget class="QLabel" name="download_courses">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>58</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Показать сохраненные курсы</string>
    </property>
   </widget>
   <widget class="QFrame" name="statusbar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 229, 229);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="status_parsing">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Неизвестен</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="show_saved_courses">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="github">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="source_site">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>393</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/change_currency.png</pixmap>
    </property>
   </widget>
   <zorder>statusbar</zorder>
   <zorder>get_combobox</zorder>
   <zorder>post_combobox</zorder>
   <zorder>label_post</zorder>
   <zorder>accept_top_courses</zorder>
   <zorder>label_get</zorder>
   <zorder>label_valute</zorder>
   <zorder>count_valute</zorder>
   <zorder>status_answer</zorder>
   <zorder>answer_marks</zorder>
   <zorder>check_marks</zorder>
   <zorder>main_icon</zorder>
   <zorder>line</zorder>
   <zorder>source_code</zorder>
   <zorder>source_site_label</zorder>
   <zorder>download_courses</zorder>
   <zorder>show_saved_courses</zorder>
   <zorder>github</zorder>
   <zorder>source_site</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
